psamm/datasource/sbml.py

```diff
@@ -750,8 +789,18 @@ def write_model(self, file, model, reactions, compounds):
             param_list = ET.SubElement(
                 kl_tag, self._sbml_tag('listOfParameters'))
 
-            lower_str = text_type(model.limits[reaction].lower)
-            upper_str = text_type(model.limits[reaction].upper)
+            if eq_id in flux_limits:
+                lower_str = text_type(flux_limits[eq_id][0])
+                upper_str = text_type(flux_limits[eq_id][1])
+            else:
+                lower_str = text_type(
+                    0
+                    if equation.direction == Direction.Forward
+                    else -model.get_default_flux_limit())
```

Comment:
This duplicates the code from above. Ideally we should only have to do this conversion of `None` values to the default flux limit in one place.